backend/src/main/java/ca/mcgill/purposeful/controller/IdeaController.java

```diff
@@ -70,6 +70,18 @@ public ResponseEntity<IdeaRequestDTO> createIdea(@RequestBody IdeaRequestDTO ide
       throw new GlobalException(HttpStatus.BAD_REQUEST, "ideaDTO is null.");
     }
 
+    // create the urls for the supporting images
+    List<String> supportingImageUrlIds = null;
```

Comment:
Do you think it would be better if this were handled in the service instead of the controller?

Reply:
I think it needs to happen here because we unpack the DTO inside the controller, before calling the service method